### PR TITLE
Beds

### PR DESCRIPTION
**Pull request by jared-pinfold**
"New Whare" process now has a + / - button for each type of bed instead of a text field.
Updated the House component to display new beds data.
Updated Rooms seed file with new beds data.

This all needs formatting now, per discussion with Grace. (different ticket on trello)

### CODE REVIEW

File: client/apis/rooms.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const baseURL = 'api/v1/rooms/'
 // }
 
 export function addRoom (room) {
+  console.log('api: ', room)
```
Comment (grace-malae) [inline client/apis/rooms.js:14]: console.log

File: client/components/Forms/AddRoomForm.jsx [hunk 2]
```diff
@@ -7,24 +7,41 @@ function AddRoomForm () {
...
+    const value = form[name]
+    if (button === 'increase') {
+      setForm({ ...form, [name]: Number(value) + 1 })
+    } else if (button === 'decrease' && value > 0) {
+      setForm({ ...form, [name]: Number(value) - 1 })
+    }
+  }
+
   function handleAddroom (e) {
+    console.log('addroom')
```
Comment (grace-malae) [inline client/components/Forms/AddRoomForm.jsx:39]: console.log

File: server/routes/rooms.js [hunk 3]
```diff
@@ -18,6 +18,7 @@ router.get('/:id', (req, res) => {
 })
 
 router.post('/', (req, res) => {
+  console.log('route: ', req.body)
```
Comment (grace-malae) [inline server/routes/rooms.js:21]: console.log

**Review by grace-malae — APPROVED** [review]
Looks great! Just checking whether the console.logs are still needed or not.

**Comment by jared-pinfold** [comment]
Console logs removed. Merging per conversation with Grace in Slack.

**Pull request merged**